src/components/PostComment.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { postCommentToArticle } from "../../api";
+
+export default function PostComment({id, setComments}) {
+    const [postStatus, setPostStatus] = useState(null);
+
+    function handleCommentPost(e) {
+        e.preventDefault();
+
+        const comment = {
+            username: e.target[0].value,
+            body: e.target[1].value
+        }
+
+
+        postCommentToArticle(id, comment)
+        .then((response) => {
```

Comment:
You're not making the most of optimistic rendering here. Please revisit the notes to see how you can give the user immediate feedback, rather than waiting for the request to finish.